src/TLuaInterpreter.cpp

```diff
@@ -309,6 +309,11 @@ int TLuaInterpreter::raiseEvent( lua_State * L )
             pE.mArgumentList.append( QString(lua_tostring( L, i )) );
             pE.mArgumentTypeList.append( ARGUMENT_TYPE_STRING );
         }
+        else if( lua_isboolean( L, i ) )
```

Comment:
Because:
* lua_isstring(...): Returns 1 if the value at the given acceptable index is a string or a number (which is always convertible to a string), and 0 otherwise. 
* lua_isnumber(...): Returns 1 if the value at the given acceptable index is a number or a string convertible to a number, and 0 otherwise.

One needs to do the isnumber check *before* the isstring one, I'm not sure but should the isboolean test be done before both of these?